samples/java-eventsourced-shopping-cart/src/main/proto/shoppingcart/shoppingcart_api.proto

```diff
@@ -56,24 +56,23 @@ service ShoppingCartService {
     component: ".domain.ShoppingCart"
   };
 
-  rpc AddItem(AddLineItem) returns (google.protobuf.Empty) {
+  rpc AddItem (AddLineItem) returns (google.protobuf.Empty) {
     option (google.api.http) = {
-      post: "/cart/{cart_id}/items/add"
-      body: "*"
-    };
+            post: "/cart/{cart_id}/items/add"
+            body: "*"
+        };
```

Comment:
Still some non-standard code formatting.

Reply:
yeah, the intellij formatter don't handle annotation properly.
I will fix by hand.